app/javascript/readingList/__tests__/readingList.test.jsx

```diff
@@ -54,7 +65,7 @@ describe('<ReadingList />', () => {
     delete global.getCsrfToken;
   });
 
-  it('should have no a11y violations', async () => {
+  it('should have no a11y violations on small screens', async () => {
```

Comment:
nitpick - this test and the "should have no a11y violations on large screens" looks to be the exact same?

Reply:
Oops, will fix that.

Reply:
done